tests/css_less/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -3276,7 +3276,7 @@ li + li {
 
 // Usage
 @import // Comment
-"@{themes}/tidal-wave.less";
+  "@{themes}/tidal-wave.less";
```

Comment:
Same situation can be for `SCSS` map too

Reply:
I'm not sure how ? I tried a single value map